app/src/main/res/layout/activity_notification.xml

```diff
@@ -26,9 +26,9 @@
                 android:layout_height="wrap_content" />
 
             <androidx.recyclerview.widget.RecyclerView
-                android:id="@+id/listView"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"/>
+              android:id="@+id/listView"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent" />
```

Comment:
Please undo stylistic changes such as this one. Thanks!

Reply:
done the required changes @nicolas-raoul